unittests/test_itertools.py

```diff
@@ -210,6 +211,56 @@ async def test_tee():
             assert await a.list(iterator) == iterable
 
 
+@multi_sync
+async def test_tee_concurrent_locked():
+    """Test that properly uses a lock for synchrinisation"""
```

Comment:
Typo: synchronisation

Reply:
You're welcome! I think this is a really nice project BTW, I think it deserves more stars. It may be a small part, but the toy event loop you've got for the tests is pretty cool, a nice example of a minimal event loop.